ngafid-core/src/main/java/org/ngafid/core/accounts/User.java

```diff
@@ -89,6 +95,28 @@ private User(ResultSet resultSet) throws SQLException {
         admin = resultSet.getBoolean(11);
         aggregateView = resultSet.getBoolean(12);
         passwordToken = resultSet.getString(13);
+        
+        // 2FA fields - handle null values for backward compatibility
+        try {
+            twoFactorEnabled = resultSet.getBoolean(14);
+        } catch (SQLException e) {
+            twoFactorEnabled = false;
+        }
+        try {
+            twoFactorSecret = resultSet.getString(15);
+        } catch (SQLException e) {
+            twoFactorSecret = null;
+        }
+        try {
+            backupCodes = resultSet.getString(16);
+        } catch (SQLException e) {
+            backupCodes = null;
+        }
+        try {
+            twoFactorSetupComplete = resultSet.getBoolean(17);
+        } catch (SQLException e) {
+            twoFactorSetupComplete = false;
+        }
```

Comment:
Cant you just set the 2FA fields to the failure fields above the try-catch and have a single try-catch block?